src/main/resources/messages/help_pl.yml

```diff
@@ -36,10 +36,10 @@ section:
 commands:
     authme.register:
         description: 'Rejestracja gracza'
-        detailedDescription: 'Rerejestracja gracza z okreslonym haslem'
+        detailedDescription: 'Rerejestracja gracza z okreslonym hasłem'
```

Comment:
Should `Rerejestracja` be `Rejestracja` or is it really correct?

Reply:
Woops, my fault. Wait a minute ;)

Reply:
Not your fault, just an opportunity to fix it since we have a pull request open :) Thanks

Reply:
Fixed.